### ISSUE TITLE
Expected behavior for glm::mat4()

### ISSUE THREAD

**Issue by Lecrapouille** [g-truc/glm#792]
Sorry I don't know well this library and its code but I confused about the different behaviors (depending on the operating system) concerning the empty constructor for `glm::mat4()`.

On my personal desktop with Debian 9.3 (as well as Ubuntu 16.04) it returns the identity matrix (what I was expected) but in Travis-CI for Darwin (OS-X) returns a matrix filled with 0 and I think this is not the expected behavior. Maybe I'm wrong but the init seems to depend on macro GLM_FORCE_CTOR_INIT and therefore can give different behavior depending on homebrew and apt-get. 

Here extracts of the log:

Homebrew:
```
==> Downloading https://homebrew.bintray.com/bottles/glm-0.9.9.0.sierra.bottle.tar.gz
==> Pouring glm-0.9.9.0.sierra.bottle.tar.gz
```

Clang version:
```
Apple LLVM version 8.1.0 (clang-802.0.42)
Target: x86_64-apple-darwin16.7.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```

**Comment by Lecrapouille** [comment]
Changing `glm::mat4()` to `glm::mat4(1.0)` fixed my issue on OS-X. But now mat4 ortho() is incompatible sign with Debian concerning the sign of mat[2][2] ! :(

**Comment by Groovounet** [comment]
The second issue is fixed in master branch thanks to report #790.

The behavior in GLM 0.9.9 is that default constructors are not initialized by default unless GLM_FORCE_CTOR_INIT is defined.

Thanks,
Christophe
